### PR TITLE
Siemens SiPass new pack

### PR DESCRIPTION
**Pull request by sharonfi99**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8756

## Description
new pack for XSIAM of Siemens SiPass 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by eepstain — APPROVED** [review]

File: Packs/Siemens_SiPass/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Siemens SiPass
+
+<~XSIAM>
+
+This pack includes Cortex XSIAM content.
+
+## Configuration on Server Side
+The log forwarding of Siemens SiPass is configured using a [file collection][here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Activate-the-Files-and-Folders-Collector).
+
+**Importent Notes**
```
Comment (ShirleyDenkberg) [inline Packs/Siemens_SiPass/README.md:10]:
```suggestion
**Important Notes**
```

File: Packs/Siemens_SiPass/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Details of the folder path on the network share containing the files that you want to monitor and upload to Cortex XSIAM.
+* Settings related to the list of files to monitor and upload to Cortex XSIAM, where the log format is either Raw (default), JSON, CSV, TSV, PSV, CEF, LEEF, Corelight, or Cisco.
```
Comment (ShirleyDenkberg) [inline Packs/Siemens_SiPass/README.md:12]: I don't understand these two bullets. Not sure what information you are trying to give.

File: Packs/Siemens_SiPass/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* Ensure that the user permissions for the network share include the ability to rename and delete files in the folder that you want to configure collection.
```
Comment (ShirleyDenkberg) [inline Packs/Siemens_SiPass/README.md:13]:
```suggestion
* Ensure that the user permissions for the network share include the ability to rename and delete files in the folder that you want to configure the collection.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

File: Packs/Siemens_SiPass/README.md (hunk 2 above)
Comment (sharonfi99) [reply Packs/Siemens_SiPass/README.md:12]: This are crucial steps of file collection, as mentioned in the file collection article 
image

**Pull request merged**